### ISSUE TITLE
Since 4.1.9 my Receiver doesn`t work in Homebridge

### ISSUE THREAD

**Issue by Stonie70** [grzegorz914/homebridge-denon-tv#188]
Hy, Since the Plugin 4.1.9. I get the error message in Homebridge Log:

Device: 192.168.20.90 Denon Receiver, prepare accessory error: TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received type number (678819940)

The Receiver doesn`t appear in Hombridge.

Please can you help me ?
Thank you very much.

**Comment by ms251699** [comment]
Same thing here:

```
Denon, prepare accessory error: TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received type number (678964830)
```

**Comment by grzegorz914** [comment]
This was fixed in 4.1.10
